metadata-service/factories/src/main/java/com/linkedin/gms/factory/telemetry/Telemetry.java

```diff
@@ -0,0 +1,142 @@
+package com.linkedin.gms.factory.telemetry;
+
+import com.mixpanel.mixpanelapi.MessageBuilder;
+import com.mixpanel.mixpanelapi.MixpanelAPI;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.exception.ExceptionUtils;
+import org.json.JSONObject;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.InputStream;
+import java.io.FileInputStream;
+import java.util.UUID;
+
+import lombok.extern.slf4j.Slf4j;
+
+import static java.lang.Boolean.parseBoolean;
+
+@Slf4j
+final class Telemetry {
+    static final String MIXPANEL_TOKEN = "5ee83d940754d63cacbf7d34daa6f44a";
+    static final String DATAHUB_FOLDER_PATH = System.getProperty("user.home") + "/.datahub";
```

Comment:
I think these paths are wrong? Are these not finalized yet? 
Ideally we should take this path as a parameter

Reply:
Fixed!